grails-app/controllers/org/broadinstitute/orsp/SearchController.groovy

```diff
@@ -122,7 +130,7 @@ class SearchController implements UserInfo {
 
     def generalReactTablesJsonSearch() {
         def user = getUser()
-        def userName = user.userName
+        def userName = user?.userName
```

Comment:
Not originally your code, true, but please please use objects instead of perpetuating the usage of `def`

Reply:
No problem. Fixed!